docs/about/all/support/getting-the-most-out-of-spryker-support.md

```diff
@@ -14,11 +14,18 @@ related:
     link: docs/about/all/support/using-the-support-portal.html
 ---
 
-We are very proud to have outstanding First Response Times (currently, they average around 35 minutes in busy weeks). However, a first response seldom immediately includes the solution to your problem. So how can we shorten Case Resolution Time?
 
-There are many of factors influencing [case resolution time](/docs/about/all/support/how-spryker-support-works.html#case-resolution-time). Some of them are determined by internal Spryker factor conditions, like availability of developer and engineering resources; others are external. One of these external factors is the question whether a case is immediately actionable. If a case contains all the information necessary for us to get working on it, it will be solved faster. How can you help us in this regard? Let's take a look at the form that is used to open a case with us:
 
-We are very proud to have outstanding [first response times](/docs/about/all/support/how-spryker-support-works.html#first-response-time) (currently they average at around 35 Minutes in busy weeks), however, a first response does seldom immediately include the solution to your problem. So how can we shorten case resolution Time?
+[Spryker Support Portal](https://support.spryker.com/s/) is the main portal for interacting with Spryker support. Spryker Customers have 3 (Standard Success) or 10 (Premier Success) licenses for the Support Portal. You can get access to the support portal by [filling out the form form](https://www.surveymonkey.com/r/XYK5R26).
+
+## Best practices for Partners and SI
```

Comment:
As I understand, we removed some permissions from partners and SIs. Why is it important to talk about that? What do they need to do in the situations where they previously created supported cases?

Reply:
We do not remove permissions, what makes you think that? They can still use the partner portal, but it is deprecated. They should start using the support portal.